lib/oxide-vpc/tests/common/mod.rs

```diff
@@ -282,10 +275,26 @@ pub fn oxide_net_setup2(
         }
     };
 
+    let v2b = Arc::new(Virt2Boundary::new());
+    v2b.set(
+        "0.0.0.0/0".parse().unwrap(),
+        TunnelEndpoint {
+            ip: "fd00:9900::1".parse().unwrap(),
+            vni: Vni::new(99u32).unwrap(),
```

Comment:
`BOUNDARY_SERVICES_VNI` would be slightly preferable, if possible.

Reply:
Done